### PR TITLE
fix(helm-chart) Fix scheduling through optional scheduler.

### PR DESCRIPTION
**Pull request by aledegano**
* Give a name to the scheduler

 * Fix required RBAC for the scheduler to work properly

### CODE REVIEW

File: helm-chart/amalthea/templates/scheduler/rbac.yaml [hunk 1]
```diff
@@ -37,9 +37,14 @@ apiVersion: rbac.authorization.k8s.io/v1
...
-    resources: ["configmaps"]
-    verbs: ["get", "list", "watch"]
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    verbs:
+    - get
+    - list
+    - watch
```
[the remark this answers is not in the document]
Comment (olevski) [reply helm-chart/amalthea/templates/scheduler/rbac.yaml:47]: I noticed that too. I think the one in `kube-system` is not needed or leftover from some troubleshooting.

I have this deployed from yesterday. But @aledegano made some changes to simplify (slim down) the rbac stuff. I want to test in a proper deployment that things work and will approve. I will also test that removing this is not causing issues. I dont think it should.

**Review by olevski — CHANGES_REQUESTED** [review]
Everything works as expected.

Just please remove the reference to this service account here: tasko-renku-amalthea-scheduler-5bb9b56ff-qtdp6

As @ableuler pointed out this is definitely not needed. As a matter of fact the service account referenced there does not even exit at all.

**Comment by olevski** [comment]
p.s. if you want to check this out this is deployed at tasko.dev.renku.ch

**Comment by aledegano** [comment]
> 

Thank you so much for testing it Tasko! I've removed the binding to the non-existing SA as suggested.

**Review by ableuler — APPROVED** [review]

**Comment by aledegano** [comment]
There are two tests failing, consistently, in the pipeline, those tests also fails locally and also fails on `main`.
Do we have any idea what is wrong with them?

**Review by olevski — APPROVED** [review]

**Pull request merged**